code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -178,7 +178,7 @@ GLOBAL_LIST_EMPTY(PDAs)
 
 /obj/item/pda/update_overlays()
 	. = ..()
-	var/mutable_appearance/overlay = new(icon)
+	var/mutable_appearance/overlay = new('icons/obj/pda.dmi')
```

Comment:
have you made sure that the AI's PDA doesn't get weird overlays now? It overrode icon before.

Reply:
nope, will take a look

Reply:
now yes